packages/syft-cli/README.md

```diff
@@ -0,0 +1,10 @@
+# Syft CLI
+
+## Development
+
+    $ pip install -e packages/syft-cli[dev]
+    $ syftcli hello
```

Comment:
A bit nitpick but I prefer putting shell commands in Github markdown like this

    ```sh
    pip install -e packages/syft-cli[dev]
    syftcli hello
    ```

instead of prefixing with `$ `. The reason is Github added the copy button to code blocks so without the `$ ` I can very quickly hit copy and paste into terminal and run immediately without having to delete the `$ `.

Though I do agree putting `$ ` makes it more clear it's a shell command so it's ok either way.

Reply:
+1 , good suggestion Kien